Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -1610,7 +1612,7 @@ namespace gdjs {
      * Called before the obstacle search.
```

Comment:
I feel like this comment should be changed, I'm having troubles figuring out what it means.